packages/compass-preferences-model/src/preferences-schema.tsx

```diff
@@ -13,10 +14,27 @@ import {
   proxyOptionsToProxyPreference,
   proxyPreferenceToProxyOptions,
 } from './utils';
+import { Link } from '@mongodb-js/compass-components';
 
 export const THEMES_VALUES = ['DARK', 'LIGHT', 'OS_THEME'] as const;
 export type THEMES = typeof THEMES_VALUES[number];
 
+const EnableDbAndCollStatsDescription: React.ReactNode = (
```

Comment:
This is element, not Component, doesn't need to be capitalized (capitalization is misleading, you can't construct it)